docs/user/reference/metrics/timing_distribution.md

```diff
@@ -435,7 +466,19 @@ assert_eq!(1, snapshot.values.len());
 
 
 </div>
-<div data-lang="JavaScript" class="tab" data-bug="1716954"></div>
+<div data-lang="JavaScript" class="tab">
+
+```Javascript
+import * as timing from "./path/to/generated/files/timing.js";
+
+const snapshot = await timing.testGetValue("<yourPingName>");
```

Comment:
This example seems a bit off. First of all, there should be a metric defined under the timing module, right? And passing the ping name explicitly in the parameter is not the normal use-case, because most metrics are only sent in a single ping passing nothing here should default to the "default" ping for this metric.

I suggest making this look a little more similar to the other examples by adding a metric under the `timing` category and removing the `"<yourPingName>"`:
```suggestion
const snapshot = await timing.serverRequest.testGetValue();
```

Reply:
Should be fixed in https://github.com/mozilla/glean/commit/f240b4a8055146e69b5360e8544297af9b0dc5df